includes/Core/Assets/Assets.php

```diff
@@ -99,6 +99,16 @@ public function register() {
 		add_action( 'admin_enqueue_scripts', $register_callback );
 		add_action( 'wp_enqueue_scripts', $register_callback );
 
+		add_action(
+			'googlesitekit_enqueue_styles',
+			function( $handle ) {
+				// Enqueue fonts.
+				$this->enqueue_fonts();
+				// Enqueue base admin screen stylesheet.
+				$this->enqueue_asset( $handle );
```

Comment:
I feel like this shouldn't do more than call `->enqueue_asset( $handle )`, it isn't really clear that this also loads fonts - the problem there of course is that it doesn't currently support loading fonts as a normal asset although I think given the tools we have now we could make that possible (the problem here is that it uses a style for AMP and a script for web). I don't want to increase the scope of this issue any farther though. I suppose we could go with this for now and then come back to it in a separate issue.

One alternative would be to provide the `Assets` instance to module classes but I don't see a great way to do that.

Thoughts @felixarntz ?

Reply:
Updated to use $assets instance created in the Plugins class.